github-actions-scripts/folioci_npmver.sh

```diff
@@ -1,39 +1,100 @@
-#!/bin/bash
+#!/usr/bin/env node
```

Comment:
It's kind of weird that this is still called `folioci_npmver.sh` now that it's a Node script, but meh. Pragmatism says not to change it. The moral is that (in accordance with standard best practice) we should never have used the `.sh` suffix in the first place.